frontend/app/actions/index.js

```diff
@@ -1,27 +1,52 @@
-export const REQUEST_BUSINESS_DETAIL = 'REQUEST_BUSINESS_DETAIL'
-export const RECEIVE_BUSINESS_DETAIL = 'RECEIVE_BUSINESS_DETAIL'
+export const BUSINESS_DETAIL_REQUEST = 'BUSINESS_DETAIL_REQUEST'
+export const BUSINESS_DETAIL_SUCCESS = 'BUSINESS_DETAIL_SUCCESS'
+export const BUSINESS_DETAIL_FAILURE = 'BUSINESS_DETAIL_FAILURE'
+export const RESET_ERROR = 'RESET_ERROR'
 
-export function requestBusinessDetail(business) {
+export function businessDetailRequest(business) {
   return {
-    type: REQUEST_BUSINESS_DETAIL,
+    type: BUSINESS_DETAIL_REQUEST,
     business
   }
 }
 
-export function receiveBusinessDetail(business, json) {
+export function businessDetailSuccess(business, json) {
   return {
-    type: RECEIVE_BUSINESS_DETAIL,
+    type: BUSINESS_DETAIL_SUCCESS,
     business,
     businessDetail: json,
   }
 }
 
+export function businessDetailFailure(error) {
+  return {
+    type: BUSINESS_DETAIL_FAILURE,
+    error: error.message || 'Something went wrong'
+  }
+}
+
+export function resetError() {
+  return {
+    type: RESET_ERROR
+  }
+}
+
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText)
+  }
+  return response
+}
+
 export function fetchBusiness(business) {
   return function (dispatch) {
-    dispatch(requestBusinessDetail(business))
-    return fetch('/api/v1/businesses/' + business['camis'])
-    .then(resp => resp.json())
-    .then(json =>
-      dispatch(receiveBusinessDetail(business, json)))
+    dispatch(businessDetailRequest(business))
+    //return fetch('/api/v1/businesses/' + business['camis'])
+    // uncomment following line to see how an error message displays in ui
+    return fetch('api/v1/businesses/2')
```

Comment:
just using this for now to debug/see how the alert works on error. will take this out before merging